rosbag2_cpp/src/rosbag2_cpp/writer.cpp

```diff
@@ -44,7 +44,15 @@ Writer::~Writer()
 void Writer::open(
   const StorageOptions & storage_options, const ConverterOptions & converter_options)
 {
-  writer_impl_->open(storage_options, converter_options);
+  writer_impl_->open(storage_options, converter_options, {});
```

Comment:
Nit:
I think it would be cleaner if the last parameter was passed explicitly. So:
```c++
open(storage_options, converter_options, CompressionOptions{});
```